Assets/GoogleVR/Demos/Scripts/DemoInputManager.cs

```diff
@@ -88,7 +92,11 @@ public enum EmulatedPlatformType {
       }
       // Message canvas will be enabled later when there's a message to display.
       messageCanvas.SetActive(false);
-#if !RUNNING_ON_ANDROID_DEVICE
+#if RUNNING_ON_IOS_DEVICE
+      // On iOS, use the Daydream controller if one's connected, and fall back to the gaze pointer if not.
+      // For notes on enabling iOS Daydream controller support, see iOSNativeControllerProvider.
+      isDaydream = GvrControllerInput.State == GvrConnectionState.Connected;
```

Comment:
When the `GVR_IOS_DAYDREAM_CONTROLLER_ENABLED` flag is omitted, `GvrControllerInput` uses `DummyControllerProvider`, so the gaze pointer is used.